.circleci/config.yml

```diff
@@ -40,6 +40,15 @@ buildCoreDNSImage: &buildCoreDNSImage
         kind load docker-image coredns
 
 jobs:
+  doc-test:
+    docker:
+      - image: circleci/golang:1.13
+    steps:
+      - checkout
+      - run: go get -u -v github.com/mmarkdown/mmark
```

Comment:
`@latest` ?

Reply:
No, it looks like it's a Go issue. Because it's doing `go get` from within the CoreDNS source, it's trying to minimize version dependency.

I guess we can do a binary fetch of the build tar file.